src/app/infobox/infobox.component.ts

```diff
@@ -47,7 +47,18 @@ export class InfoboxComponent implements OnInit {
       }
     });
   }
-
+  getQuery(q) {
```

Comment:
can you please explain me the root cause of this issue, due to which variable we are facing this issue?